pkg/ee/cmd/kubermatic-installer/shared.go

```diff
@@ -35,6 +39,10 @@ var (
 		Usage: "Namespace to use for created resources",
 		Value: "kubermatic",
 	}
+	pauseSeedsFlag = cli.BoolFlag{
```

Comment:
I think we are using this command only for this migration, would not be easier to have this defaulted to `true`?

Reply:
Good point